vulnfeeds/cmd/combine-to-osv/run_combine_to_osv_convert.sh

```diff
@@ -41,5 +41,5 @@ echo "Override"
 gcloud --no-user-output-enabled storage rsync "gs://${INPUT_BUCKET}/osv-output-overrides/" $OSV_OUTPUT
 
 echo "Begin syncing output to GCS bucket ${OUTPUT_BUCKET}"
-gcloud --no-user-output-enabled storage rsync "$OSV_OUTPUT" "gs://${OUTPUT_BUCKET}/osv-output/" --checksums-only -c --delete-unmatched-destination-objects -q
+gsutil -q -m rsync -c -d "${OSV_OUTPUT}" "gs://${OUTPUT_BUCKET}/osv-output/"
```

Comment:
Use the full flags here to make it easier to understand what they do

Reply:
I checked `gsutil help rsync`, they don't really have full flags for most options. also this command is the same as Debian one, so I think it's safe to use.